Meshtastic/Helpers/MeshPackets.swift

```diff
@@ -706,7 +706,7 @@ func telemetryPacket(packet: MeshPacket, connectedNode: Int64, context: NSManage
 							subtitle: "AKA \(telemetry.nodeTelemetry?.user?.shortName ?? "UNK")",
 							content: "Time to charge your radio, there is \(telemetry.batteryLevel)% battery remaining.",
 							target: "nodes",
-							path: "meshtastic://nodes?nodenum=\(telemetry.nodeTelemetry?.num ?? 0)"
+							path: "meshtastic:///nodes?nodenum=\(telemetry.nodeTelemetry?.num ?? 0)"
```

Comment:
I'm not familiar with the extra forward slash. Is that an apple deeplink navigation scheme?

Reply:
No, this is just the way to separate the `path` component of the URL per RFC-1738

The deeplink urls have a custom scheme, but no user/host/port, and only have a path at the end.

`meshtastic` is the scheme, the `://` is separating the scheme from the host (which is empty), and the third slash denotes the start of the path.

Its kind of pedantic, but it enables me to parse the URL more easily using `Foundation.URLComponents` and by looking solely at the `path` component, instead of having to combine the `host` and `path` and do additional logic.